sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -179,6 +179,22 @@ private PollerFlux(Duration pollInterval,
             (context, response) -> Mono.error(ex), context -> Mono.error(ex));
     }
 
+    /**
+     * Updates the poll interval for this poller.
```

Comment:
Should we add a text indicating that - by default the next poll delay is the value returned from last poll `PollResponse`'s `getRetryAfter()`, if the returned value is null or zero then the poller uses the provided `pollInterval`.

Reply:
Improved the documentation for this method. Note that the poll interval can never be null or zero. The constructor as well as the setter method both validate the input pollInterval.